api/src/lib/stripe.js

```diff
@@ -0,0 +1,24 @@
+import Stripe from 'stripe'
+
+export const stripe = new Stripe(process.env.STRIPE_SK)
```

Comment:
Your Stripe client lives here now. Think of it just like the db in `api/src/lib/db.js`—instantiate it once and import it from here.